gui/healthbar.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+
+class HealthBar():
+    def __init__(self, x, y, hp, max_hp):
+        self.x = x
+        self.y = y
+        self.hp = hp
+        self.max_hp = max_hp
+
+
+        def draw(self, hp):
+            self.hp = hp
+            ratio = self.hp / self.max_hp
+            pygame.draw.rect(screen, red, (self.x, self.y, 150, 20))
+            pygame.draw.rect(screen, green, (self.x, self.y, 150 * ratio, 20))
```

Comment:
Colors could be not predefined, it's better to do it as color_of_background_bar, and color_of_front_bar